routes/api/v1/event_exports.js

```diff
@@ -353,13 +269,12 @@ exports.plugin = {
           results.forEach(_renameAndClearFields);
 
           if (request.query.format && request.query.format === "csv") {
+            const { events } = flattenEventJSON(results);
+            csv_results = convertToCSV(events, request.query.use_renav);
 
-            const flattenJSON = _flattenJSON(results);
-            const csvHeaders = _buildCSVHeaders(flattenJSON);
-            
-            const csv_results = await parseAsync(flattenJSON, { fields: csvHeaders });
-
-            return h.response(csv_results).code(200);
+            return h.response(csv_results)
+              .header('Content-Length', csv_results.length)
```

Comment:
There's no way you have to manually set `Content-Length`... but then again this *is* JavaScript...

Reply:
I don't think it's valid HTTP to not send C-L with a body...